### DISCUSSION TITLE
Copilot Chat unnecessary censorship "Make bullet list" for hilighted text

### DISCUSSION THREAD

**Discussion by Aciid** [community/community#70315] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hello, 

I tried making my multiline text into html bullet list by hilighting the rows in VSCode Copilot Chat and using prompt "Make bullet list" the answer got filtered. "Oops, your response got filtered. Vote down if you think this shouldn't have happened."

Using specific prompt "Transform into HTML bullet list" worked.

**Comment by Akash1134** [comment, marked as answer]
Hi @Aciid ,

You may encounter this error unexpectedly while using GitHub Copilot Chat. We are aware of this issue and working to fix it. You can [report the error here](https://github.com/orgs/community/discussions/56134) so we can have consolidated feedback.
